### PR TITLE
Added STATIC_ROOT and changed docker.dev

### PR DESCRIPTION
**Pull request by BassCoder2808**
- Intended to solve #511 

@goneall have changed `dockerfile.dev` and added the `collectstaic` command there. This should work and all the static files should get collected. Can you try that once and do let me know if anything else needs to be done in this.

### CODE REVIEW

File: dockerfile.dev [hunk 1]
```diff
@@ -29,5 +29,6 @@ EXPOSE 8000
 
 # Run migrations and start the development server on port 8000
 CMD python src/manage.py migrate && \ 
+    python src/manage.py collectstatic && \
```
Comment (goneall) [inline dockerfile.dev:32]: Please make the same change to the dockerfile.prod

**Review by goneall — CHANGES_REQUESTED** [review]
Thanks @BassCoder2808 - I like this approach.  Just one change requested.

File: dockerfile.dev (hunk 1 above)
Comment (BassCoder2808) [reply dockerfile.dev:32]: @goneall was going through the `dockerfile.prod` but couldn't find how we are starting the django server.
wanted to ask like do we manually run the commands while deploying or how is it?

File: src/src/settings.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 API_DIR = os.path.join(BASE_DIR,'api')
 TEMPLATE_DIR = os.path.join(APP_DIR, 'templates')
 STATIC_PATH = os.path.join(APP_DIR,'static')
+STATIC_ROOT = os.path.join(APP_DIR,'static')
```
Comment (rtgdk) [inline src/src/settings.py:22]: Let's make it 
```
STATIC_ROOT = STATIC_PATH
```

**Review by rtgdk — CHANGES_REQUESTED** [review]

File: dockerfile.dev (hunk 1 above)
Comment (goneall) [reply dockerfile.dev:32]: It is run under [supervisor](http://supervisord.org/).  The last line in the docker file starts it up.

[supervisor_api.conf](https://github.com/spdx/spdx-online-tools/blob/main/supervisor_api.conf) is the config file.

Supervisor starts up [gunicorn](https://gunicorn.org/) to run a [wsgi](https://wsgi.readthedocs.io/en/latest/what.html) script which run Django via wsgi.

Here's some background on how that is setup: https://docs.djangoproject.com/en/5.0/howto/deployment/wsgi/

Comment (BassCoder2808) [reply dockerfile.dev:32]: Thanks @goneall for this, I will make the appropriate changes now

**Review by rtgdk — APPROVED** [review]

**Review by goneall — APPROVED** [review]
Updates look good - Thanks @BassCoder2808

**Pull request merged**